cnf-tests/testsuites/e2esuite/test_suite_test.go

```diff
@@ -128,12 +131,18 @@ var _ = BeforeSuite(func() {
 	}
 	_, err = testclient.Client.Namespaces().Create(context.Background(), ns, metav1.CreateOptions{})
 	Expect(err).ToNot(HaveOccurred())
+
+	// Expect(err)... handled internally
+	// note this intentionally does NOT set the infra we depends on the configsuite for this
+	numaserialconf.Config = numaserialconf.SetupFixture()
```

Comment:
Not blocking, but this looks a bit awkward.
A module function initializing a module global, called from outside. I think having it all self-contained in the module would be better, especially considering that Config is not used anywhere here.

Reply:
You are right. We can clean up the API a bit. Since we will need to have more updates here after the bulk of this changes goes in (this PR), we will have https://github.com/openshift-kni/numaresources-operator/pull/257 sorted out and merged by that time, and we will clean up this like you suggested.